keps/sig-node/1769-memory-manager/kep.yaml

```diff
@@ -0,0 +1,44 @@
+title: Memory Manager
+kep-number: 1769
+authors:
+  - "@bg-chun"
+  - "@cezaryzukowski"
+  - "@cynepco3hahue"
+owning-sig: sig-node
+participating-sigs:
+status: implemented
```

Comment:
This should be implementable. A KEP status will only be implemented once the feature has graduated to GA.  :smile:

Reply:
Many thanks, updated 😄